src/DICOM.jl

```diff
@@ -158,21 +159,23 @@ function pixeldata_parse(st, sz, vr, dcm)
         dtype = UInt16
     end
     if !is(dcm,false)
+	# (0028,0010) defines number of rows
         f = lookup(dcm, (0x0028,0x0010))
         if !is(f,false)
-            xr = f.data[1]
+            xr = f.data[1][1]
```

Comment:
I will go to @ultradian for any additional input, but I believe this has to do with some of the changes in julia array indexing since v5.0.

`f.data[1]` actually returns a a tuple/array containing the scalar integer of interest here. So the double index would return that scalar integer.